integration-tests/integration/signIn/signIn.spec.js

```diff
@@ -14,72 +14,66 @@ context('Login functionality', () => {
     cy.task('stubKeyworkerMigrated')
   })
 
-  it('Root (/) redirects to the auth login page if not logged in', () => {
-    cy.task('stubLoginPage')
+  it('Root (/) redirects to the auth sign in page if not signed in', () => {
+    cy.task('stubSignInPage')
     cy.visit('/')
     cy.url().should('include', 'authorize')
     cy.get('h1').should('contain.text', 'Sign in')
   })
 
-  it('Login page redirects to the auth login page if not logged in', () => {
-    cy.task('stubLoginPage')
-    cy.visit('/login')
```

Comment:
I think we should keep a `/login` test, as it is still there (albeit just a redirect)

Reply:
yeah - your right - I'll have a look at that.